ast/misc.go

```diff
@@ -463,6 +463,8 @@ func (n *BinlogStmt) Accept(v Visitor) (Node, bool) {
 // See https://dev.mysql.com/doc/refman/5.7/en/commit.html
 type CommitStmt struct {
 	stmtNode
+	// CompletionType overwrites system variable `completion_type` within transaction
+	CompletionType uint8
```

Comment:
Please define a type alias instead of using `uint8` here, e.g.

```go
type CompletionType uint8
const (
     CompletionTypeDefault CompletionType = 0
     CompletionTypeRelease                = 2
     ...
)
```

Reply:
got it, sorry for my poor code